contributors/design-proposals/cloudprovider-storage-metrics.md

```diff
@@ -49,37 +49,46 @@ Since we are interested in count(or rate) and latency percentile metrics of API
 the external Cloud Provider - we will use [Histogram](https://prometheus.io/docs/practices/histograms/) type for
 emitting these metrics.
 
-We will be using `HistogramVec` type so as we can attach dimensions at runtime. Whenever available
-`namespace` will reported as a dimension with the metric.
+We will be using `HistogramVec` type so as we can attach dimensions at runtime.
```

Comment:
Should we almost keep track of errors? E.g. the error count in the GCE proposed implementation...